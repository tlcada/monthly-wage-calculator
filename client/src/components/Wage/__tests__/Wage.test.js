import React from 'react'
import { shallow, mount } from 'enzyme'
import sinon from 'sinon'
import toJson from 'enzyme-to-json'
import Wage from '../Wage'

test('Wage component should render as expected', () => {
  const component = shallow(<Wage />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
