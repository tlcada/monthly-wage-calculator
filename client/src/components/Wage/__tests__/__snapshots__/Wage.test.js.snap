// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wage component should render as expected 1`] = `
<div>
  <Table
    bordered={false}
    bsClass="table"
    condensed={false}
    hover={true}
    responsive={false}
    striped={true}
  >
    <thead>
      <tr>
        <th>
          Person Name
        </th>
        <th>
          Wage
        </th>
        <th>
          Month
        </th>
      </tr>
    </thead>
    <tbody />
  </Table>
</div>
`;
